ops-tests/component/test_vtysh_ct_bgp_neighbor.py

```diff
@@ -875,6 +875,25 @@ def verify_no_neighbor_ttl_security_hops_peergroup(step):
         foundpeergroupcfg is False
     )
 
+def verify_neighbor_weight_cmd(step):
+    step("Verifying neighbor weight/no neighbor weight commands")
+    switch = dutarray[0]
+    weight = "65534"
+    nbr_cfg_string = "neighbor " + bgp2_router_id+" weight " + weight
+
+    switch("end")
+    switch("conf t")
+    switch("router bgp %s" % bgp1_asn)
+    switch("neighbor %s weight %s" % (bgp2_router_id, weight))
+    dump = switch("do show running-config")
+    assert nbr_cfg_string in dump
+    switch("no neighbor %s weight" % bgp2_router_id)
+    dump = switch("do show running-config")
+    assert nbr_cfg_string not in dump
+    switch("neighbor %s weight %s" % (bgp2_router_id, weight))
+    switch("no neighbor %s weight %s" % (bgp2_router_id, weight))
```

Comment:
First command is no neighbor X weight 65534, second command is no neighbor X weight.